packages/runtime/src/utils/load.ts

```diff
@@ -194,6 +196,60 @@ async function loadEntryNode({
     });
 }
 
+async function loadEntryReactNative({
```

Comment:
since you have added `loadEntry` hook , you can add `react-native` support by providing runtime plugin .

like adding a plugin named `react-native-plugin` [here](https://github.com/module-federation/core/tree/main/packages/runtime/src/plugins) , and then add exports in package.json . 

Users can use the runtimePlugin on demand